README.md

```diff
@@ -176,11 +189,11 @@ example4 = runM . runReader "hello" . runState 0 $ do
 
 ### Required compiler extensions
 
-To use effects, you'll typically need `-XFlexibleContexts`.
+To use effects, you’ll typically need `-XFlexibleContexts`.
```

Comment:
I don’t think this is true any more.

Reply:
I believe this is true; this morning I encountered

```
    • Non type-variable argument
        in the constraint: Control.Effect.Sum.Member Random sig1
      (Use FlexibleContexts to permit this)
```

Reply:
It’s [no longer necessary for the smart constructors](https://github.com/fused-effects/fused-effects/pull/320), nor generally for use of them.

Reply:
Beautiful! Will fix.